addons/languages/thai/pack/src/main/res/xml/thai_semialign.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<Keyboard xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:ask="http://schemas.android.com/apk/res-auto"
+    android:keyWidth="7.692307%p"
+    android:keyHeight="@integer/key_normal_height"
+    ask:showPreview="false"
+    ask:autoCap="false">
+    <Row>
+        <Key android:codes="3613" android:keyLabel="ฝ" ask:keyOutputTyping="ฝ" ask:isShiftAlways="true" ask:shiftedCodes="3622" ask:shiftedKeyLabel="ฦ" ask:shiftedKeyOutputTyping="ฦ" android:keyEdgeFlags="left" />
```

Comment:
What does this mean? in regards to `isShiftAlways=true`

Reply:
By ASK's default sticky shift does not apply to symbols or other non-letters. But Thai keyboards need sticky shift to apply to every key, so I introduce `isShiftAlways` to allow overriding the method ASK use to determine whether sticky shift should apply or not.